packages/design-system/src/new-ui/Menubar/Menubar.tsx

```diff
@@ -238,23 +228,42 @@ const MenubarShortcut = ({
     />
   );
 };
-MenubarShortcut.displayname = "MenubarShortcut";
+MenubarShortcut.displayName = "MenubarShortcut";
+
+type MenubarType = {
+  Root: typeof MenubarRoot;
+  Menu: typeof MenubarPrimitive.Menu;
+  Trigger: typeof MenubarTrigger;
+  Content: typeof MenubarContent;
+  Item: typeof MenubarItem;
+  Separator: typeof MenubarSeparator;
+  Label: typeof MenubarLabel;
+  CheckboxItem: typeof MenubarCheckboxItem;
+  RadioGroup: typeof MenubarPrimitive.RadioGroup;
+  RadioItem: typeof MenubarRadioItem;
+  Portal: typeof MenubarPrimitive.Portal;
+  SubContent: typeof MenubarSubContent;
+  SubTrigger: typeof MenubarSubTrigger;
+  Group: typeof MenubarPrimitive.Group;
+  Sub: typeof MenubarPrimitive.Sub;
+  Shortcut: typeof MenubarShortcut;
+};
 
-export const Menubar = {
+export const Menubar: MenubarType = {
   Root: MenubarRoot,
-  Menu: MenubarMenu,
+  Menu: MenubarPrimitive.Menu,
   Trigger: MenubarTrigger,
   Content: MenubarContent,
   Item: MenubarItem,
   Separator: MenubarSeparator,
   Label: MenubarLabel,
   CheckboxItem: MenubarCheckboxItem,
-  RadioGroup: MenubarRadioGroup,
+  RadioGroup: MenubarPrimitive.RadioGroup,
   RadioItem: MenubarRadioItem,
-  Portal: MenubarPortal,
+  Portal: MenubarPrimitive.Portal,
   SubContent: MenubarSubContent,
   SubTrigger: MenubarSubTrigger,
-  Group: MenubarGroup,
-  Sub: MenubarSub,
+  Group: MenubarPrimitive.Group,
+  Sub: MenubarPrimitive.Sub,
   Shortcut: MenubarShortcut,
 };
```

Comment:
@thewbuk Why we need to cast type here, I think you don't need to change this type?